### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by cknabs** [cknabs/EUvsDisinfo#5]
The accessibility of the app should be improved. This includes: 
- Making the HTML accessible using attributes
- Choosing an accessible color scheme for the graphs and the app layout
- Minimizing used ressources and app load time
- Checking accessibility metrics on push or pull requests, e.g. using @AccessLint 

Ideally, the app would target A11Y support A or AA.
